tools/lsp/README.md

```diff
@@ -75,9 +75,11 @@ go to the *User Server Settings*, and  enter the following in the text area: (**
 ```json
 {
   "servers": {
-	"SixtyFPS": {
-	  "command": ["/home/YOUR_USER/.cargo/bin/sixtyfps-lsp"],
-	  "highlightingModeRegex": "SixtyFPS"	}
+    "SixtyFPS": {
+      "path": ["${ENV:HOME}/.cargo/bin", "%{ENV:USERPROFILE}/.cargo/bin"],
```

Comment:
I can see %{ENV:..} in the kate docs but not ${ENV:..}. Are you sure it’s correct to use both?

Reply:
I think so. I took this from [Kate's configuration for rust-analyzer](https://invent.kde.org/utilities/kate/-/blob/33a748a44d90299210703f0f0cbce88e14cb8a5c/addons/lspclient/settings.json#L104).

Reply:
Oh, that uses `%` for both... hmm...

Reply:
I asked in the Kate Matrix room and `%` is correct.